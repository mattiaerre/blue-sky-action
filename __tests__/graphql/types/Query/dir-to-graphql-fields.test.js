const { join } = require('path');
const dirToGraphQLFields = require('../../../../src/graphql/types/Query/dir-to-graphql-fields.js');

describe('dirToGraphQLFields', () => {
  it('to match snapshot', () => {
    const dirPath = join(
      __dirname,
      '../../../../src/graphql/types/Query/fields'
    );
    expect(dirToGraphQLFields(dirPath)).toMatchSnapshot();
  });
});
