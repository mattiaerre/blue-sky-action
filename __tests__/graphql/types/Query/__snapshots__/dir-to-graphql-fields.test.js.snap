// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`dirToGraphQLFields to match snapshot 1`] = `
Object {
  "articles": Object {
    "args": Object {
      "q": Object {
        "type": "String",
      },
    },
    "resolve": [Function],
    "type": "[Article]",
  },
  "buyPrice": Object {
    "args": Object {
      "currencyPair": Object {
        "type": "String",
      },
    },
    "resolve": [Function],
    "type": "Price",
  },
  "currencies": Object {
    "resolve": [Function],
    "type": "[Currency]",
  },
  "exchangeRates": Object {
    "args": Object {
      "currency": Object {
        "type": "String",
      },
    },
    "resolve": [Function],
    "type": "ExchangeRates",
  },
  "ip": Object {
    "resolve": [Function],
    "type": "String",
  },
  "marathons": Object {
    "resolve": [Function],
    "type": "[Marathon]",
  },
  "name": Object {
    "resolve": [Function],
    "type": "String",
  },
  "sources": Object {
    "args": Object {
      "q": Object {
        "type": "String",
      },
    },
    "resolve": [Function],
    "type": "[Source]",
  },
  "spotPrices": Object {
    "args": Object {
      "currency": Object {
        "type": "String",
      },
    },
    "resolve": [Function],
    "type": "[Price]",
  },
  "topHeadlines": Object {
    "args": Object {
      "source": Object {
        "type": "String",
      },
    },
    "resolve": [Function],
    "type": "[Article]",
  },
}
`;
